src/test/tasks/tasks.yaml

```
@@ -41,7 +41,13 @@ tasks:
       - cmd: echo "I'm set from setVariables - ${ACTION_VAR}"
       - cmd: echo "I'm set from a runner var - ${REPLACE_ME}"
       - cmd: echo "I'm set from a --set var - ${REPLACE_ME}"
+      - cmd: echo "I'm set from a --set var - $REPLACE_ME"
       - cmd: echo "I'm set from a new --set var - ${UNICORNS}"
+      - cmd: echo "I'm set automatically - MARU=${MARU}"
```

```suggestion
      - cmd: echo "I'm set automatically - MARU=${MARU}"
      - cmd: MARU=false echo "${MARU}"
```
We should add a test and remove the `cmd` templating and just pull from env

---

(the removal of that templating should be marked as a breaking change in the PR desc)